pkg/api/v1/types.go

```diff
@@ -2140,6 +2140,10 @@ type PodSpec struct {
 	// If specified, the pod's scheduling constraints
 	// +optional
 	Affinity *Affinity `json:"affinity,omitempty" protobuf:"bytes,18,opt,name=affinity"`
+	// If specified, the pod will be dispatched by specified scheduler.
+	// If not specified, the pod will be dispatched by default scheduler.
+	// +optional
+	SchedulerName string `json:"schedulername,omitempty" protobuf:"bytes,19,opt,name=schedulername"`
```

Comment:
This is wrong - this should have been `schedulerName`

Please ensure this is fixed pre 1.6.0 - this is a release blocker.